tasks/test.js

```diff
@@ -75,45 +75,19 @@ module.exports = function (grunt) {
 
   grunt.registerTask('test:ui', [
     'checkPlugins',
-    'run:testEsServer',
-    'run:funcTestServer',
```

Comment:
This was the only reference to the `run:funcTestServer` task, can we remove it now?

Reply:
I think so